Makefile

```diff
@@ -1,24 +1,34 @@
 TAG := $(shell git rev-parse --short HEAD)
 DIR := $(shell pwd -L)
 DIR := $(shell pwd -L)
-SDCLI_VERSION :=v1.5
-SDCLI=docker run --rm -v "$(DIR):$(DIR)" -w "$(DIR)" asecurityteam/sdcli:$(SDCLI_VERSION)
+GO_IMAGE ?= golang:1.24
+GODOCKER = docker run --rm -v "$(DIR):$(DIR)" -w "$(DIR)" $(GO_IMAGE)
+LINT_IMAGE ?= golangci/golangci-lint:latest
```

Comment:
request:

Can we pin this to a specific version? I think pulling latest can cause some build stability problems and we can probably.

Also ditto with up above, could we extract this "lint image" to just be a separate Dockerfile that our Makefiles will reference? 

IMO it will make it easier for renovate to do updates and it creates a better separation of concerns between the Makefile and the Dockerfiles:
- Docker files handle container config and versioning)
- Makefile is used for lightweight commands and environment variable overrides for basic config options we need for build tools like the unit test coverage file paths etc....

Reply:
+1